cli/cmd/install-sp.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"text/template"
+
+	"github.com/linkerd/linkerd2/cli/installsp"
+	"github.com/spf13/cobra"
+)
+
+type installSPConfig struct {
+	Namespace string
+}
+
+func newCmdInstallSP() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install-sp [flags]",
```

Comment:
Consider a more verbose command name for clarity like `install-profiles` or `install-service-profiles`.

Reply:
Ditto. I like `install-service-profiles` as it fully describes what the command is going to do when I run it.

Reply:
I agree these names better convey what's going on. I just implemented it, and the length feels out of sync with the rest of the commands (`install-sp` is already tied with `completion` for longest command).

Given that rest of our subcommands are all extremely short, and probably require the user to run `linkerd` or `linkerd subcmd --help` to understand what each does anyway, I'm inclined to leave it as-is. For comparison, kubectl's longest cmd is `rolling-update`, and most are much shorter and a single word.